modules/Lending.py

```diff
@@ -73,10 +74,13 @@ def init(cfg, api1, log1, data, maxtolend, dry_run1, analysis, notify_conf1):
 
     sleep_time = sleep_time_active  # Start with active mode
 
-    if notify_conf['enable_notifications'] and notify_conf['notify_summary_minutes']:
+    if notify_conf['enable_notifications']:
```

Comment:
This change will make the scheduler thread run even if no notifications are configured.